ui/app/components/confirm-page-container/confirm-page-container-content/confirm-page-container-content.component.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import { Tabs, Tab } from '../../tabs'
+import {
+  ConfirmPageContainerSummary,
+  ConfirmPageContainerError,
+  ConfirmPageContainerWarning,
+} from './'
+
+export default class ConfirmPageContainerContent extends Component {
+  static propTypes = {
+    action: PropTypes.string,
+    title: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    titleComponent: PropTypes.func,
+    subtitle: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    hideSubtitle: PropTypes.bool,
+    errorMessage: PropTypes.string,
+    summaryComponent: PropTypes.node,
+    detailsComponent: PropTypes.node,
+    dataComponent: PropTypes.node,
+    identiconAddress: PropTypes.string,
+    nonce: PropTypes.string,
+    warning: PropTypes.string,
+  }
+
+  renderContent () {
+    const { detailsComponent, dataComponent } = this.props
+
+    if (detailsComponent && dataComponent) {
+      return this.renderTabs()
+    } else {
+      return detailsComponent || dataComponent
+    }
+  }
+
+  renderTabs () {
+    const { detailsComponent, dataComponent } = this.props
+
+    return (
+      <Tabs>
```

Comment:
It might be nice if the tabs component had an API that allowed:
```
<Tabs
  tabs={
    'Details': detailsComponent,
    'Data': dataComponent,
  }
/>
```

Reply:
I was actually trying to avoid this so that the elements would be more HTML semantic:
```
<Tabs>
  <Tab />
  <Tab />
</Tabs>
```